docs/_src/tutorials/tutorials/6.md

```diff
@@ -116,13 +116,13 @@ See [their docs](https://milvus.io/docs/v1.0.0/milvus_docker-cpu.md) for more de
 
 
 ```python
-!pip install git+https://github.com/deepset-ai/haystack.git#egg=farm-haystack[milvus]
+# !pip install git+https://github.com/deepset-ai/haystack.git#egg=farm-haystack[milvus]
```

Comment:
The tutorial 6 changes from https://github.com/deepset-ai/haystack/pull/2148 made it in here. We shouldn't do that. Could you please remove them?